test/e2e/plugin_test.go

```diff
@@ -0,0 +1,49 @@
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPlugins(t *testing.T) {
+	testCases := []struct {
+		image  string
+		report string
+		err    error
+	}{
+		{
+			image:  "docker.io/library/nginx:1.23",
+			report: "./testdata/fake_report.json",
+			err:    fmt.Errorf("Error: unsupported osType FakeOS specified\n"), // nolint:revive
+		},
+		{
+			image:  "docker.io/library/alpine:3.14.0",
+			report: "./testdata/invalid_report.json",
+			err:    fmt.Errorf("Error: error running scanner fake: exit status 1\n"), // nolint:revive
+		},
```

Comment:
can we add a valid case too? we can add a super simple `valid_report.json` 

for example, something like:
```
{
    "OSType": "alpine",
    "OSVersion": "3.7.3",
    "Arch": "amd64",
    "Packages": [
        {
            "Name": "musl",
            "InstalledVersion": "1.1.18-r3",
            "FixedVersion": "1.1.18-r4",
            "VulnerabilityID": "CVE-2019-14697"
        }
    ]
}
```